app/components/Input.tsx

```diff
@@ -104,31 +104,17 @@ export const LabelText = styled.div`
   display: inline-block;
 `;
 
-export type Props = Omit<React.HTMLAttributes<HTMLInputElement>, "onChange"> & {
+export type Props = React.InputHTMLAttributes<
+  HTMLInputElement | HTMLTextAreaElement
+> & {
```

Comment:
These types had to get changed a bit to play nicely with `react-hook-form`

I thin the impactful change is around the `innerRef` the rest was making typescript happy while I was here